### PR TITLE
modif on foo.md

### PR DESCRIPTION
**Pull request by Anthyko**
I changed your foo ma dude

### CODE REVIEW

**Comment by raphael-batlle** [comment]
How could you do this to me? :(

File: foo.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-Hey you !
+Hey i modified this, awesome ! 
 We’re trying git at School !
-With the fabulous @Nek-  <3.
```
Comment (Anthyko) [inline foo.md:3]: Fabulous is not enough for him

File: foo.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
-Hey you !
+Hey i modified this, awesome !
```
Comment (raphael-batlle) [inline foo.md:1]: It's better, thanks dude

**Review by raphael-batlle — APPROVED** [review]
It's ok

**Pull request merged**